src/main/java/com/yelp/nrtsearch/tools/nrt_utils/legacy/incremental/RestoreIncrementalCommand.java

```diff
@@ -27,6 +27,7 @@
 import com.amazonaws.services.s3.transfer.TransferManagerBuilder;
```

Comment:
The `RestoreIncrementalCommand` is a frozen legacy command. It should not be updated so that is continues to work with v0 clusters. We will create new versions of there commands after we finish the backend structural changes.

Note this only applies the legacy commands, the state command below is fine to update.